ml-agents/mlagents/trainers/agent_processor.py

```diff
@@ -134,8 +133,8 @@ def add_experiences(
                         agent_id=agent_id,
                         next_obs=next_obs,
                     )
-                    # This will eventually be replaced with a queue
-                    self.trainer.process_trajectory(trajectory)
+                    for _traj_queue in self.trajectory_queues:
```

Comment:
nit: don't use leading underscores for local variables.

Reply:
Fixed